packages/components/input/src/input.vue

```diff
@@ -371,18 +371,22 @@ const onceInitSizeTextarea = createOnceInitResize(resizeTextarea)
 
 const setNativeInputValue = () => {
   const input = _ref.value
-  if (!input || input.value === nativeInputValue.value) return
-  input.value = nativeInputValue.value
+  const formatterValue = props.formatter
+    ? props.formatter(nativeInputValue.value)
+    : nativeInputValue.value
+  if (!input || input.value === formatterValue) return
+  input.value = formatterValue
 }
 
 const handleInput = async (event: Event) => {
   recordCursor()
 
   let { value } = event.target as TargetElement
 
+  const parseValue = props.parser ? props.parser(value) : value
+
   if (props.formatter) {
-    value = props.parser ? props.parser(value) : value
-    value = props.formatter(value)
```

Comment:
Regarding handleInput, I think that as long as this line of code is deleted, the values returned by the two emit events should be the same.

Reply:
It should be to restore this handleInput, and then delete `value = props.parser ? props.parser(value) : value`.
Your current existence should be a problem, because the precondition of the parser is formatter, and you have done the processing of the parser first.

Reply:
I have finished. But failing of 'PR Docs Build / Build windows-latest LTS (pull_request)' ?